front/app/modules/commercial/content_builder/admin/components/ContentBuilderTopBar/index.tsx

```diff
@@ -148,6 +147,23 @@ const ContentBuilderTopBar = ({
             </>
           )}
         </Box>
+        {!isNilOrError(locales) && selectedLocale && locales.length > 1 && (
```

Comment:
We can remove the check for `!isNilOrError(locales)` here since we are handling that on line 72.

Reply:
Thanks, Edwin 🙌 This is indeed a good point and I will definitely fix it. But generally, if the comment is not blocking (which in this case I don't believe it is), it's a good practice to approve the PR and avoid another round trip of reviews.

Reply:
That is a good point, approving now